pom.xml

```diff
@@ -137,6 +137,7 @@
         <flexmark.version>0.62.2</flexmark.version>
         <flink.version>1.16.1</flink.version>
         <flink.module.scala.suffix></flink.module.scala.suffix>
+        <flink.python.module.scala.suffix></flink.python.module.scala.suffix>
```

Comment:
can we avoid introducing this property by leveraging the maven transitive dependency mechanism?

Reply:
The `flink-python` module is individual. There are no flink modules that depend on it. After we drop the 1.14 and 1.15 version, we can remove this property.